scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -432,6 +435,34 @@ static struct task_ctx *lookup_task_ctx(struct task_struct *p)
 	return taskc;
 }
 
+/* HACK opcode imported wholesale from linux/prctl.h */
+#define PR_SET_NAME 15
+
+SEC("ksyscall/prctl")
+int BPF_KSYSCALL(prctl_enter, int option, unsigned long arg2,
+	unsigned long arg3, unsigned long arg4, unsigned long arg5)
```

Comment:
We're already attaching to `tp_task_rename()`, so no need to create another one.